components/common/Article.tsx

```diff
@@ -49,6 +53,11 @@ const ArticleContainer = styled(Link)`
   width: 100%;
   height: 17rem;
 
+  &.hover {
+    transform: translateY(0.8rem);
+    transition: 0.3s ease-in-out;
```

Comment:
요거 ease-in-out 말구 ease-out으로 통일...로 알고 있는데 아닌가요?

```suggestion
    transition: 0.3s ease-out;
```